apis/infix-en_GB/extensions/jdk8/atrium-api-infix-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/jdk8/optionalAssertions.kt

```diff
@@ -0,0 +1,56 @@
+@file:Suppress("JAVA_MODULE_DOES_NOT_READ_UNNAMED_MODULE" /* TODO remove once https://youtrack.jetbrains.com/issue/KT-35343 is fixed */)
+
+package ch.tutteli.atrium.api.infix.en_GB.jdk8
+
+import ch.tutteli.atrium.api.infix.en_GB.Empty
+import ch.tutteli.atrium.api.infix.en_GB.Present
+import ch.tutteli.atrium.api.infix.en_GB.jdk8.creating.optional.PresentWithCreator
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.domain.builders.optional
+import java.util.*
+
+/**
+ * Expects that the subject of the assertion (an [Optional]) is empty (not present).
+ *
+ * Shortcut for more or less something like `feature(Optional<T>::isEmpty) { toBe(true) }`
+ * depends on the underlying implementation though.
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.11.0
+ */
+infix fun <T : Optional<*>> Expect<T>.toBe(@Suppress("UNUSED_PARAMETER") Empty: Empty) = addAssertion(ExpectImpl.optional.isEmpty(this))
```

Comment:
please split into two lines
```suggestion
infix fun <T : Optional<*>> Expect<T>.toBe(@Suppress("UNUSED_PARAMETER") Empty: Empty) = 
    addAssertion(ExpectImpl.optional.isEmpty(this))
```